releasenotes/releasenotes.rst

```diff
@@ -26,6 +26,23 @@ The default FHIR version of the `public Firely Server endpoint <https://server.f
 
 .. _vonk_releasenotes_490:
```

Comment:
Please add a new link for 5.0.0

Reply:
Wait, why are we not releasing this in 4.10?